uefi_app/SbsaAvsMain.c

```diff
@@ -510,23 +510,32 @@ ShellAppMainsbsa (
   Print(L"\n      ***  Starting GIC tests ***  \n");
   Status |= val_gic_execute_tests(g_sbsa_level, val_pe_get_num());
 
+#ifndef ONLY_SBSA_RULE_TESTS
```

Comment:
What is "ONLY_SBSA_RULE_TESTS" mean? Why with this macro will skip "Timer tests" and other tests, and default with this build option in SbsaAvs.inf?